servicecontrol/monitoring-instances/installation/index.md

```diff
@@ -7,33 +7,45 @@ tags:
 
 The ServiceControl installation file consists of an embedded MSI bootstrapper EXE and an embedded MSI. This installation can be executed standalone or via the [Platform Installer](/platform/installer/). The installation package include a utility to manage the installation, upgrade and remove of ServiceControl services, including Monitoring instances. This utility is launched as the final step in the installation process and is also available via the Windows Start Menu.
 
-
 ## Prerequisites
 
 The ServiceControl Installation has the following prerequisites:
 
- * [Microsoft .NET 4.5.2 Runtime](https://www.microsoft.com/en-us/download/details.aspx?id=42643)
+* [Microsoft .NET 4.5.2 Runtime](https://www.microsoft.com/en-us/download/details.aspx?id=42643) for Version 1 and 2
```

Comment:
Question - with SC, aren't we supporting the latest version only? I.e. why would someone go and install v1 or v2 when v3 is out?

Reply:
@tmasternak given that there is no data backwards compatibility concerns perhaps we can stick with only documenting the current state of affairs? (I know it's different for SC)